### ISSUE TITLE
Scribble V3

### ISSUE THREAD

**Issue by yedhink** [udai1931/scribble-by-udai-gupta#88]
- [x] Pagination in Articles Dashboard ( Already done in #87 )
- [x] Seed the DB w/ services ( Already done in #87 )
- [x] Publish/Unpublish later feature
- [x] Clickable & blue colored title in table
- [x] Research for prop drilling and thorough code check
- [x] Thorough UI check
- [x] More tests to make 100% coverage

https://www.loom.com/share/b41c7845df194a73bf351a47282630d0

@udai1931 _a Work on this after #87 . 

This is an epic. Raise sub issues and work on it.
